test/libyul/EVMCodeTransformTest.cpp

```diff
@@ -53,6 +53,7 @@ TestCase::TestResult EVMCodeTransformTest::run(ostream& _stream, string const& _
 	settings.optimizeStackAllocation = m_stackOpt;
 	YulStack stack(
 		EVMVersion{},
+		std::nullopt,
```

Comment:
```suggestion
		nullopt,
```
Also below.

Reply:
I wonder when clang will raise an error about all of those beyond ``std::move`` (which was already arbitrary)

Reply:
Why would it? Isn't `using namespace std` a common thing? I'm actually surprised that it's even warning about `std::move`.

Reply:
But it *does* it for ``std::move`` :-).

Reply:
Also in my book ``using namespace std`` is still horrible style, but let's not go there, I'll remove them ;-).

Reply:
Well, I also kinda prefer not using `using namespace std` but since we do that's more consistent :)